dtfs-central-api/src/v1/controllers/portal/deal/update-deal.controller.js

```diff
@@ -91,47 +77,43 @@ const updateDeal = async (dealId, dealChanges, user, existingDeal, routePath) =>
       }
 
       let originalDealDetails;
+      let originalDealEligibility;
+
+      const update = {
```

Comment:
Can we explain this change in code comment? what processes are modifying deal at same time and how we avoid issue.
Things like multiple `updatedAt` updates can happen in future too.

Reply:
Good shout, have added.